learn-data-structures-by-building-a-shopping-cart/index17.js

```diff
@@ -50,20 +50,17 @@ class ShoppingCart {
   }
   
   applyDiscount(amount) {
-    return this.discountPercentage / 100 * amount
-  }
-}
-
-/*
-Notice that the returned amount of appliyDiscount function has a lot of numbers after the decimal.
+    /*
+    Notice that the discount money has more than 2 numbers after the decimal point.
```

Comment:
Notice that the expected discount has more than 2 numbers after the decimal point.
    This is common to see when working with numbers in JavaScript.
    There is in-built method called `toFixed()` that can help us.
    This method takes a number as a parameter for the amount of digits that you want to appear after the decimal point.
    It will also round up the number if necessary.
    Wrap the formula in parentheses and chain `toFixed(2)` to the end of it.